test/utils.ts

```diff
@@ -113,3 +120,16 @@ export async function pause(ms: number) {
     }, ms),
   );
 }
+
+export function getUserKeypairFromFile(keypairPath) {
+  const keypairDataString = readFileSync(path.resolve(keypairPath), {
+    encoding: 'utf-8',
+  });
+
+  const keypairData = keypairDataString
+    .substring(1, keypairDataString.length - 1)
+    .split(',')
+    .map((item) => +item);
+
+  return Keypair.fromSecretKey(new Uint8Array(keypairData));
```

Comment:
Just pleas use JSON parse here instead of implementing this manually after you read the file.
`require` does this all in one step and will even keep the parsed JSON in the `require.cache`.

So this whole function would be be something along the lines of:

```js
const arr = require(keypairPath);
const u8Array = Uint8Array.from(arr);
return Keypair.fromSecretKey(u8Array);
```